.github/ISSUE_TEMPLATE/falsepositive.yml

```diff
@@ -0,0 +1,112 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+name: False Positive Report
+description: File a Phishing report
+labels: ["False Positive"]
+assignees:
+  - mitchellkrogza
+  - funilrys
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand being listed on a Phishing Database like this can be frustrating and embarrassing for many web site owners.
+
+        The first step is to remain calm.
+        
+        The second step is to rest assured one of our maintainers will address your issue as soon as possible.
+
+        Please make sure you have provided as much information details as possible to help speed up the process.
+
+  - type: textarea
+    id: record
+    attributes:
+      label: Domain/URL/IP(s) you believes NOT to be Phishing
+      description: |
+        Please list any domains and links listed here which you believe are a false positive.
+        
+        Please warn with "NSFW" where applicable.
```

Comment:
May be better:

```
Please warn the community with "NSFW" - if applicable.
```

Reply:
I read `where` more as the **mandatory** than I do with a `IF` >>Yeah right *IF* I fell like doing that today<<

Reply:
Fair.